src/features/hotspots/setup/HotspotSetupConfirmLocationScreen.tsx

```diff
@@ -77,7 +77,7 @@ const HotspotSetupConfirmLocationScreen = () => {
       locationNonceLimit: onboardingRecord.maker.locationNonceLimit,
       makerAddress: onboardingRecord.maker.address,
     }
-    Location.loadLocationFeeData(feeParams).then(setFeeData)
+    loadLocationFeeData(feeParams).then(setFeeData)
```

Comment:
`feeParams.nonce` needs to be set from the hotspot's nonce instead of 0? In the homepage, all the details of a hotspot including  `nonce` should be pulled already.

Reply:
What about using null safe chain operator in `account.balance.integerBalance` since `account.balance` can be null.
https://github.com/helium/helium-js/blob/e713a707ab9304765a718359383dd83624950fd6/packages/http/src/models/Account.ts#L33

Reply:
> What about using null safe chain operator in `account.balance.integerBalance` since `account.balance` can be null. https://github.com/helium/helium-js/blob/e713a707ab9304765a718359383dd83624950fd6/packages/http/src/models/Account.ts#L33

https://github.com/NebraLtd/maker-starter-app/blob/87d854f905d1083b50fd92ad4365f76871b0ae3c/src/features/hotspots/setup/HotspotSetupConfirmLocationScreen.tsx#L69

it will return if account.balance is null before it gets to the mentioned line